src/main/scala/io/getclump/package.scala

```diff
@@ -0,0 +1,15 @@
+package io
+
+package object getclump {
+
+  private[getclump] implicit val executionContext = scala.concurrent.ExecutionContext.global
+
+  private[getclump]type Promise[T] = scala.concurrent.Promise[T]
+  private[getclump] val Promise = scala.concurrent.Promise
+
+  private[getclump]type Future[+T] = scala.concurrent.Future[T]
+  private[getclump] val Future = scala.concurrent.Future
+
+  def blockOn[T](future: Future[T]) =
```

Comment:
don't forget, this should be private visibility

also in my opinion this should be named `awaitResult` to align with existing naming